### ISSUE TITLE
No Acess for Fine_Tuning Documentation

### ISSUE THREAD

**Issue by Roopesh-Bharatwaj-K-R** [NASA-IMPACT/hls-foundation-os#22]
Hi,

Hope you are doing well, I'm Looking to do a Fine-tuning of the Prithvi model but I could not find the steps and examples
once I looked in the repo "fine-tuning-examples/configs/sen1floods11.py" PAGE NOT FOUND error 

kindly suggest and help me through the fine-tuning process.

I'm a bit confused about the Data Engineering Part: how to handle the data,  as the data is in GeoTTIF Format. how can we label it manually? 

What is the expected input format for fine-tuning the Prithvi?

Thanks and best Regards,
Roopesh Bharatwaj K R

**Comment by CarlosGomes98** [comment]
Hi,

I am not sure what directory you are referring to. The Readme provides example commands to run fine-tuning on any of the 3 provided examples in the configs folder.

Labels are expected to be a GeoTiff with one channel, where each pixel location has a label. E.g. you may look at the Sen1Floods11 example dataset for what the framework expects.

**Comment by Roopesh-Bharatwaj-K-R** [comment]
Thanks for the information and update to the readme! it's useful.  I have a doubt, I can see in the config file we need to set the path of ( pre-trained weight ),  Not sure where to get it from.


`# Model -> sent1floods11_config.py`
  `# Need to Set path to pretrained backbone weights`
`pretrained_weights_path = "<path to pretrained weights>"`

best

**Comment by longdaothanh** [comment]
> Thanks for the information and update to the readme! it's useful. I have a doubt, I can see in the config file we need to set the path of ( pre-trained weight ), Not sure where to get it from.
> 
> `# Model -> sent1floods11_config.py` `# Need to Set path to pretrained backbone weights` `pretrained_weights_path = "<path to pretrained weights>"`
> 
> best

Have you found a solution for this? Thanks
